packages/atomic/src/pages/examples/ipx.html

```diff
@@ -189,12 +189,32 @@
+            icon='<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 409.6 103.7"><path d="M164.7 36.2l-4.9 5.2c-3.4-3.3-7-5.2-11.9-5.2-8.5 0-14.9 6.7-14.9 16.1s6.4 16.1 14.9 16.1c4.6 0 9-1.9 12.2-5.2l4.6 5.2c-4 4.6-9.7 7-16.4 7-14 0-23.1-10-23.1-23.1 0-13.4 9.1-23.4 23.1-23.4 6.3-.1 12.3 2.6 16.4 7.3zm54.7 16.1c0 13.1-10 23.1-23.4 23.1-13.7 0-23.7-10-23.7-23.1 0-13.4 10-23.4 23.7-23.4 13.4 0 23.4 10 23.4 23.4zm-39.2 0c0 9.4 6.7 16.1 15.8 16.1 8.8 0 15.5-6.7 15.5-16.1s-6.7-16.1-15.5-16.1c-9.1 0-15.8 6.7-15.8 16.1zm72.1 22.2h-9.2l-18.8-44.4h8.5l14.9 36.8 14.9-36.8h8.2l-18.5 44.4zM318.8 52v3h-35c.9 9.1 7.3 13.7 14.9 13.7 5.1.1 10.1-1.8 14-5.2l4 5.2c-5.5 5.2-11.6 6.8-18.3 6.8-13.4 0-22.8-9.1-22.8-23.1 0-13.7 9.4-23.4 22.2-23.4 12.2-.1 21 9.6 21 23zm-34.9-3.4h27.4c-.9-7.6-6.1-12.8-13.4-12.8-7.9.1-12.8 5.2-14 12.8zm92.1 3.7c0 13.1-10 23.1-23.4 23.1-13.7 0-23.7-10-23.7-23.1 0-13.4 10-23.4 23.7-23.4 13.3 0 23.4 10 23.4 23.4zm-39.2 0c0 9.4 6.7 16.1 15.8 16.1 8.8 0 15.5-6.7 15.5-16.1s-6.7-16.1-15.5-16.1c-9.2 0-15.8 6.7-15.8 16.1z" fill="#323256"/><path d="M390.8 31.9V43h-1.9V31.9h-3.5v-1.8h9v1.8h-3.6zM407.7 43v-8.6l-3 6.2h-1.4l-3-6.2V43h-2V30.2h2l3.7 8 3.7-8h1.9V43h-1.9z" opacity=".87" fill="#323256"/><path d="M88.1 14.9C79.5 6.4 68.3 1.2 56.2 0c-.9 0-1.6.8-1.6 1.8 0 .4.1.7.4 1l18.8 18.5c0 .3 0 .3-.3.3-5.2-3.8-11.3-6.1-17.6-6.7-.6-.1-1.1.3-1.2.9 0 .3.1.6.3.9l12.8 12.8s0 .3-.3 0c-3.5-2.2-7.4-3.6-11.6-4-.6 0-.9.6-.6 1.2l21.6 21.6c.6.6 1.5 0 1.2-.6-.3-4.1-1.7-8.1-4-11.6-.3 0 0-.3 0 0l12.5 12.2c.6.6 1.8 0 1.8-.9-.7-6.4-3-12.6-6.7-17.9-.3 0 0-.3 0 0L100.6 48c.5.6 1.3.7 1.9.3.5-.3.7-.9.5-1.5-1.1-12.1-6.4-23.4-14.9-31.9z" fill="#00adff"/><path d="M14.9 14.9C23.6 6.5 34.8 1.2 46.8 0c.9 0 1.6.6 1.7 1.5 0 .4-.1.9-.5 1.2L29.5 21.3c-.3.3 0 .3 0 .3 5.3-3.8 11.5-6.1 17.9-6.7.9 0 1.5 1.2.9 1.8L35.6 29.5c-.3 0 0 .3 0 0 3.5-2.2 7.4-3.5 11.6-4 .6 0 1.2.6.6 1.2L26.1 48.3c-.6.6-1.2 0-1.2-.6.4-4.1 1.9-8.1 4.3-11.6 0 0-.3-.3-.3 0L16.7 48.3c-.9.6-2.1 0-1.8-.9.6-6.5 2.9-12.7 6.7-17.9 0 0-.3-.3-.3 0L2.7 48c-1.2 1.2-2.7.3-2.7-1.2 1.2-12 6.5-23.2 14.9-31.9z" fill="#f05245"/><path d="M14.9 88.8c8.5 8.6 19.8 13.9 31.9 14.9.8.1 1.5-.4 1.6-1.2.1-.5-.1-.9-.4-1.3L29.5 82.4c-.3 0 0-.3 0 0 5.3 3.8 11.5 6.1 17.9 6.6.9 0 1.5-1.2.9-1.8L35.6 74.5c-.3-.3 0-.3 0-.3C39 76.5 43 78 47.1 78.4c.6 0 1.2-.9.6-1.2L26.1 55.6c-.6-.6-1.2-.3-1.2.6.4 4.1 1.9 8.1 4.3 11.6 0 0-.3.3-.3 0L16.7 55.6c-.9-.6-2.1-.3-1.8.9.6 6.4 2.9 12.5 6.7 17.6 0 .2-.1.3-.3.3L2.7 55.6c-.7-.6-1.8-.4-2.3.3-.2.3-.3.6-.4 1 1.1 12 6.4 23.3 14.9 31.9z" fill="#1cebcf"/><path d="M88.1 88.8c-8.5 8.7-19.8 14-31.9 14.9-1.2.3-2.1-1.5-1.2-2.4l18.8-18.8s0-.3-.3 0c-5.2 3.8-11.3 6.1-17.6 6.7-.6.1-1.1-.4-1.2-1 0-.3.1-.6.3-.9l12.8-12.8c0-.3 0-.3-.3-.3-3.4 2.4-7.4 3.9-11.5 4.2-.5-.1-.8-.5-.7-.9 0-.1 0-.2.1-.3L77 55.6c.6-.6 1.5-.3 1.2.6-.3 4.1-1.7 8.1-4 11.6-.3 0 0 .3 0 0l12.5-12.2c.4-.4 1.1-.4 1.5 0 .2.2.3.6.3.9-.7 6.3-3 12.4-6.7 17.6-.3.3 0 .3 0 .3l18.8-18.8c.9-.9 2.7 0 2.4 1.2-1 12.1-6.3 23.4-14.9 32z" fill="#ffe300"/></svg>'
+          />
```

Comment:
It's the coveo logo. I'm not to sure whether to keep it here.

Reply:
🤔 should we add it to our known icons maybe?

Reply:
We could add it to the assets perhaps yes, but optional, shipping it to users is not really important

Reply:
I'll keep it here since it's just a simple example.